README.md

```diff
@@ -54,85 +72,70 @@ The Optimism Immunefi program offers up to $2,000,042 for in-scope critical vuln
+└── <a href="./ufm-test-services">ufm-test-services</a>: Runs a set of tasks to generate metrics
 </pre>
 
-## Branching Model
+## Development and Release Process
 
-### Active Branches
+### Overview
 
-| Branch          | Status                                                                           |
-| --------------- | -------------------------------------------------------------------------------- |
-| [master](https://github.com/ethereum-optimism/optimism/tree/master/)                   | Accepts PRs from `develop` when intending to deploy to production.                  |
-| [develop](https://github.com/ethereum-optimism/optimism/tree/develop/)                 | Accepts PRs that are compatible with `master` OR from `release/X.X.X` branches.                    |
-| release/X.X.X                                                                          | Accepts PRs for all changes, particularly those not backwards compatible with `develop` and `master`. |
+Please read this section if you're planning to fork this repository, or make frequent PRs into this repository.
 
-### Overview
+### Production Releases
+
+Production releases are always tags, versioned as `<component-name>/v<semver>`.
+For example, an `op-node` release might be versioned as `op-node/v1.1.2`, and an L1 smart contract release might be versioned as `contracts-L1/v1.0.0`.
+Release candidates are versioned in the format `op-node/v1.1.2-rc.1`.
+We always start with `rc.1` rather than `rc`.
+
+Tags of the form `v<semver>`, such as `v1.1.4`, indicate releases of all Go code only, and **DO NOT** include smart contracts.
+This naming scheme is required by Golang.
+In the above list, this means these `v<semver` releases contain all `op-*` components, and exclude all `contracts-*` components.
 
-This repository generally follows [this Git branching model](https://nvie.com/posts/a-successful-git-branching-model/).
-Please read the linked post if you're planning to make frequent PRs into this repository.
+`op-geth` embeds upsteam geth’s version inside it’s own version as follows: `vMAJOR.GETH_MAJOR GETH_MINOR GETH_PATCH.PATCH`.
+Basically, geth’s version is our minor version.
+For example if geth is at `v1.12.0`, the corresponding op-geth version would be `v1.101200.0`.
+*Note that we pad out to three characters for the geth minor version and two characters for the geth patch version.
+Since we cannot left-pad with zeroes, the geth major version is not padded.
 
-### Production branch
+The full set of components that have releases are:
 
-The production branch is `master`.
-The `master` branch contains the code for latest "stable" releases.
-Updates from `master` **always** come from the `develop` branch.
+- `op-node`
+- `op-batcher`
+- `op-proposer`
+- `op-challenger`
+- `contracts-L1`
+- `contracts-L2`
+
+All other components and packages should be considered development components only and do not have releases.
```

Comment:
Is this accurate?

Reply:
EAS runs in production and is heavily utilized by retroPGF and Base. It should have releases, we technically could put it in `L2` since its a predeploy

Reply:
Resolving this in favor of the other similar thread https://github.com/ethereum-optimism/optimism/pull/8731#discussion_r1434522701